src/rebar3_hex_version.erl

```diff
@@ -0,0 +1,30 @@
+%%% ===================================================================
+%%% Version utils
+%%% ===================================================================
```

Comment:
Not unlike rebar3_hex_app, this is just a module that will hold pure functions related to version parses, incrementing, and formatting.